specification/quantum/data-plane/Microsoft.Quantum/preview/2021-05-06-preview/quantum.json

```diff
@@ -0,0 +1,757 @@
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/SasUriResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/restError"
+            }
+          }
+        }
+      }
+    },
+    "/v1.0/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Quantum/workspaces/{workspaceName}/quotas": {
+      "get": {
+        "operationId": "Quotas_List",
+        "description": "List quotas for the given workspace.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/workspaceNameParameter"
+          }
+        ],
+        "x-ms-examples": {
+          "Get a list of quotas for a particular quantum workspace.": {
+            "$ref": "./examples/quotaList.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/QuotaList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/restError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "JobDetailsList": {
+      "description": "List of job details.",
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/JobDetails"
+          }
+        },
+        "count": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Total records count number."
+        },
+        "nextLink": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Link to the next page of results."
+        }
+      }
+    },
+    "JobDetails": {
+      "description": "Job details.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The job id."
+        },
+        "name": {
+          "type": "string",
+          "description": "The job name. Is not required for the name to be unique and it's only used for display purposes."
+        },
+        "containerUri": {
+          "type": "string",
+          "description": "The blob container SAS uri, the container is used to host job data."
+        },
+        "inputDataUri": {
+          "type": "string",
+          "description": "The input blob SAS uri, if specified, it will override the default input blob in the container."
+        },
+        "inputDataFormat": {
```

Comment:
Is there a known set of values for this format making it eligible to define an extensible enum?

Reply:
Out of scope for this PR since I didn't modify this area.